users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -1,31 +1,27 @@
 from typing import List, Dict, Optional, Any
 
-
```

Comment:
Why do you remove these lines? Did you check that with black?

Reply:
Done